spec-2021x.md.html

```diff
@@ -420,6 +420,14 @@
  0.0  | 25.3245 | 3.32435 | 0.16801 | -1.27393 | -4.85967
  1.0  | 21.5473 | 3.82987 | 0.19823 | -1.97760 | -4.32054
  [Table [sheen-lambda-fit]: Values for $\Lambda$ curve fitting. Parameters are interpolated according to $P = (1-r_b)^2 P_{0.0} + (1-(1-r_b)^2) P_{1.0}$ [#CK17].]
+ 
+If performance is a concern, due to the computation cost of the term above, we propose an alternative visibility term, given in [#NP13]:
+
+\begin{equation}
+M_b(\mathbf{v},\mathbf{l}; r_b) = \frac{D_b(\mathbf{h},r_b)}{4 ((\mathbf{v} \cdot \mathbf{n}) + (\mathbf{l} \cdot \mathbf{n}) - (\mathbf{v} \cdot \mathbf{n}) (\mathbf{l} \cdot \mathbf{n}))}
+\end{equation}
+
+The visual differences between the main model and the alternative model can be observed on the following ShaderToy: https://www.shadertoy.com/view/wl3SWs.
```

Comment:
That's a good alternative for the complicated visibility term, thanks for the proposal. Would be great if you could add side-by-side comparison images to show the difference, basically screenshots from the ShaderToy with a short description. Is the sheen BSDF still energy conserving if the Ashikhmin visibility term is used?

Besides, we may have to provide a separate dataset for directional albedo. I can take care of that.

Reply:
Sure, I'll add some images. I'll make two rows of images with growing roughness with a 0.1 step

The sheen BSDF is not energy conservative with the new term if I recall correctly, it adds a little bit of energy for low roughness values and leaks a little bit on high roughness values. I'll make a white furnace test for it and add the result. I should have the time during this week.

We are definitively going to need another directional albedo, but should also add a numerical fit for it. Anyone going for that alternative term will most likely be concerned about the sampler quantities too.